### PR TITLE
feat: owner profile

### PR DESCRIPTION
**Pull request by zhpeng811**
Show owner detail in detail pages (except for owner)

This PR contains #141, #142 and #145, might want to get those reviewed first

<details> <summary> Detail page </summary>
<p>

Screenshot_1606262138

</p>
</details>

<details> <summary> On click </summary>
<p>

Screenshot_1606262145

</p>
</details>

### CODE REVIEW

**Comment by c25vdw** [comment]
My intention of creating a fragment container is for injecting whatever button and however many of them as we want with customizable title and body and onclick and a hook. 
<details> <summary>For this implementation there's a styling issue for accepted detail.</summary>

image
</details>
<details> <summary>what I recommend is, you could just do something similar to this :point_down: and call the inflate wherever appropriate </summary>

image
</details>

File: app/src/main/res/layout/owner_detail_button.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="3dp"
+    android:padding="6dp"
+    android:layout_marginBottom="8dp"
+    android:id="@+id/owner_detail_button"
```
Comment (c25vdw) [inline app/src/main/res/layout/owner_detail_button.xml:9]: we may delete this file?

**Review by c25vdw — APPROVED** [review]
lgtm

File: app/src/main/res/layout/owner_detail_button.xml (hunk 1 above)
Comment (zhpeng811) [reply app/src/main/res/layout/owner_detail_button.xml:9]: yes, I'll make sure it's deleted before it's merged

**Review by kdvuong — APPROVED** [review]
lgtm

**Pull request merged**